pkg/addons/assets/efa-device-plugin.yaml

```diff
@@ -57,6 +57,7 @@ spec:
                       - g4dn.metal
                       - g5.48xlarge
                       - hpc6a.48xlarge
+                      - hpc7g.16xlarge
```

Comment:
V - lets update with all hpc7g types here since they all have EFA. Add hpc7g.8xlarge, hpc7g.4xlarge.

Reply:
Can do! I didn’t want to add them before being able to test. Any updates on if it might be possible to have an ARM build for the plugin?

Reply:
Done! And per our discussion, I can offer to do a PR to add these images here https://github.com/aws-samples/efa-device-plugin-helm/blob/main/aws-efa-k8s-device-plugin after we finish up here.

Reply:
okay opened, albeit we should not merge it! https://github.com/aws-samples/efa-device-plugin-helm/pull/1